figma-tokens.json

```diff
@@ -2,6 +2,10 @@
   "core": {
     "colors": {
       "neutral": {
+        "0": {
+          "value": "transparent",
+          "type": "colors"
```

Comment:
Well done @washucode 
is possible to change this type to color instead of colors